scvi/external/scbasset/_module.py

```diff
@@ -68,9 +91,12 @@ def __init__(
         activation_fn: Optional[Callable] = None,
     ):
         super().__init__()
-        self.dense = nn.Linear(in_features, out_features, bias=not batch_norm)
-        self.batch_norm = nn.BatchNorm1d(out_features) if batch_norm else nn.Identity()
-        self.activation_fn = activation_fn if activation_fn is not None else nn.GELU()
+        self.dense = _Linear(in_features, out_features, bias=not batch_norm)
+        # batch norm with Keras default epsilon
+        self.batch_norm = (
+            nn.BatchNorm1d(out_features, eps=0.001) if batch_norm else nn.Identity()
```

Comment:
there should be another batchnorm in the conv block

Reply:
wdym?

Reply:
ohh ok